Dockerfiles/gophermart/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# syntax=docker/dockerfile:1
+
+# Build the application from source
+FROM golang:1.20
+
+WORKDIR /app
+
+COPY ./go.mod ./go.mod
+COPY ./go.sum ./go.sum
+RUN go mod download
+
+COPY ./bin ./bin
+COPY ./cmd/gophermart ./cmd/gophermart
+COPY ./db/gophermart ./db/gophermart
+COPY ./docs/gophermart ./docs/gophermart
+COPY ./internal/gophermart ./internal/gophermart
+COPY ./internal/common ./internal/common
+COPY ./*.md ./
```

Comment:
Why?

Reply:
`COPY ./docs/gophermart ./docs/gophermart` - for future swagger.
`COPY ./internal/gophermart ./internal/gophermart` - for comple bin (our server)
`COPY ./internal/common ./internal/common` - for compile bin (utils for verify jwt)
`COPY ./*.md ./` - because no one forbids us to put documentation into the container ))